isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -783,6 +784,785 @@ namespace Isis {
+      std::vector<double> coefDEC;
+      std::vector<double> coefTWI;
+
+      int nPositionCoefficients = m_solveSettings->numberCameraPositionCoefficientsSolved();
+      int nPointingCoefficients = m_solveSettings->numberCameraAngleCoefficientsSolved();
+
+      // Indicate if we need to obtain default position or pointing values
+      bool useDefaultPosition = false;
+      bool useDefaultPointing = false;
+      // Indicate if we need to use default values when not solving twist
+      bool useDefaultTwist = !(m_solveSettings->solveTwist());
+
+      // If we aren't solving for position, set the number of coefficients to 1 so we can output the
+      // instrumentPosition's center coordinate values for X, Y, and Z
+      if (nPositionCoefficients == 0) {
+        nPositionCoefficients = 1;
+        useDefaultPosition = true;
+      }
+      // If we arent' solving for pointing, set the number of coefficients to 1 so we can output the
+      // instrumentPointing's center angles for RA, DEC, and TWI
+      if (nPointingCoefficients == 0) {
+        nPointingCoefficients = 1;
+        useDefaultPointing = true;
+      }
+
+      // Force number of position and pointing parameters to each be 3 (X,Y,Z; RA,DEC,TWI)
+      // so we can always output a value for them
+      int nPositionParameters = 3 * nPositionCoefficients;
+      int nPointingParameters = 3 * nPointingCoefficients;
+      int nParameters = nPositionParameters + nPointingParameters;
+
+      coefX.resize(nPositionCoefficients);
+      coefY.resize(nPositionCoefficients);
+      coefZ.resize(nPositionCoefficients);
+      coefRA.resize(nPointingCoefficients);
+      coefDEC.resize(nPointingCoefficients);
+      coefTWI.resize(nPointingCoefficients);
+
+      if (m_instrumentPosition) {
+        if (!useDefaultPosition) {
+          m_instrumentPosition->GetPolynomial(coefX, coefY, coefZ);
+        }
+        // Use the position's center coordinate if not solving for spacecraft position
+        else {
+          const std::vector<double> centerCoord = m_instrumentPosition->GetCenterCoordinate();
+          coefX[0] = centerCoord[0];
+          coefY[0] = centerCoord[1];
+          coefZ[0] = centerCoord[2];
+        }
+      }
+
+      if (m_instrumentRotation) {
+        if (!useDefaultPointing) {
+          m_instrumentRotation->GetPolynomial(coefRA, coefDEC, coefTWI);
+        }
+        // Use the pointing's center angles if not solving for pointing (rotation)
+        else {
+          const std::vector<double> centerAngles = m_instrumentRotation->GetCenterAngles();
+          coefRA[0] = centerAngles[0];
+          coefDEC[0] = centerAngles[1];
+          coefTWI[0] = centerAngles[2];
+        }
+      }
+
+      // for convenience, create vectors of parameters names and values in the correct sequence
+      std::vector<double> finalParameterValues;
+      QStringList parameterNamesList;
+      QStringList correctionUnitList;
+
+        QString str("%1(%2)  ");
```

Comment:
The following block is misaligned.

Reply:
The function this block was in has been deleted because it's no longer needed.